src/Extract.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr  9 19:33:06 2019
+
+This is the Extract Module for my ETL code.
+It must satisfy the following contract:
+    1. Accept a QUERY DATE, S3 resource instance, boolean
+       telling function whether its being run locally or by
+       an AWS Lambda, local temp space, and a bucket name as inputs
+    2. read sample-extract-output.csv from the top
+       level folder in my S3 Bucket (mybucket1219)
+    3. write the unmodified file to
+       s3://$BUCKET/sample-project/extract/$QUERY_DATE/${QUERY_DATE}-extract-output.csv
+       such that $BUCKET == mybucket1219
+
+@author: zackkingbackup
+"""
+
+def extract_function(QUERY_DATE,s3_resource,LOCAL=False,
```

Comment:
Why are  some of the parameter names upper-cased?  I'd prefer all lower-case.  Upper case makes me think they are constants or that they are environment variables.

Reply:
No reason. I have no problem going all lowercase.